Sources/XcodeProj/Objects/Project/PBXProjEncoder.swift

```diff
@@ -468,4 +469,23 @@ final class PBXProjEncoder {
             navigatorGroups.values.forEach { $0.children = $0.children.sorted(by: sort) }
         }
     }
+
+    private func sortProjectReferences(for projects: [PBXProject], outputSettings: PBXOutputSettings) {
+        guard outputSettings.projReferenceFormat == .xcode else {
+            return
+        }
+
+        for project in projects {
+            /// The project references are sorted alphabetically based on the name of the project it's being referenced.
+            project.projectReferences = project.projectReferences.sorted(by: { lhs, rhs in
```

Comment:
@Speakus in which scenario a project contains more than one project reference. Would you mind adding a fixture under `Fixtures` so that we can add some tests?